hotsos/plugin_extensions/openstack/service_network_checks.py

```diff
@@ -149,12 +150,12 @@ def _get_router_iface_mtus(self):
     @summary_entry('router-port-mtus', get_min_available_entry_index() + 8)
     def summary_router_port_mtus(self):
         """ Provide a summary of ml2-ovs router port mtus. """
-        project = getattr(self, 'neutron')
-        if not (project and project.bind_interfaces):
+        bind_interfaces = self.project_helpers.neutron.bind_interfaces
```

Comment:
aa, neutron can be "None" 
Quck note, I'm not fan of putting everywhere "null checks" but since it was already there it makes sense to keep it.
eg.
bind_interfaces = self.project_helpers.neutron.bind_interfaces if self.project_helpers.neutron is not None else None

Reply:
you still have to split this over several lines though so i feel like you end up with the same thing?

Reply:
re: neutron can be None, I don't believe this is the case anymore hence why i removed the check.

Reply:
neutron, can't be None in current code but if you create object like:
        self.project_helpers = OSTProjectHelpers(NovaBase(), None, OctaviaBase())
it will be.
Resolving this as there might not be a point for writing a checks for code which does not exists

Reply:
Hmm it looks like I can't resolve the comments